shell/app/modules/msp/monitor/trace-insight/pages/trace-querier/trace-querier.tsx

```diff
@@ -197,10 +202,21 @@ const TraceInsightQuerier = () => {
               />
             </FormItem>
           </Col>
-          <Col span={3}>
+          <Col span={6}>
             <Button type="primary" loading={isRequestTraceFetching} onClick={handleRequestTrace}>
               {i18n.t('msp:request')}
             </Button>
+            <Button className="ml-4" danger onClick={() => setModalVis(true)}>
+              {i18n.t('common:reset')}
+            </Button>
+            <Modal
```

Comment:
Modal.confirm is better

Reply:
![image](https://user-images.githubusercontent.com/3955437/128655729-200139d7-17cb-4df3-92f6-55185e2fdddd.png)
Use Popconfirm in such condition.